src/modules/WizardsWardrobeBanking.lua

```diff
@@ -116,17 +116,13 @@ function WWB.DepositSetup(zone, pageId, index)
 	WWB.MoveItems(gearTable, bankBag)
 end
 
-function WWB.DepositAllSetups(skipCurrentCharacter)
-	local bankBag = GetBankingBag()
-	if WW.DISABLEDBAGS[bankBag] then return end
-	
-	local itemLocationTable = WW.GetItemLocation()
+function WWB.GetAccountSavedGear(skipCurrentCharacter)
+  local itemLocationTable = WW.GetItemLocation()
 	local preGearTable = {}
-	WW.Log(GetString(WW_MSG_DEPOSIT_ALL), WW.LOGTYPES.NORMAL, "FFFFFF")
-	
+
 	for characterId, characterSv in pairs(WizardsWardrobeSV.Default[GetDisplayName()]) do
-		if characterId ~= "$AccountWide" then		
-			for zoneTag, setupPages in pairs(characterSv.setups) do
+		if (skipCurrentCharacter and characterSv["$LastCharacterName"] ~= GetUnitName("player")) and characterId ~= "$AccountWide" then

```

Comment:
If `skipCurrentCharacter` is false, this function will never go any further. Which would also break the existing features that used this logic